_scripts/Dockerfile.gvim

```diff
@@ -24,7 +25,8 @@ ENV PATH=/vim/bin:$PATH
 RUN rm ~/.netrc
 
 ARG GO_VERSION
-RUN curl -sL https://dl.google.com/go/${GO_VERSION}.linux-amd64.tar.gz | tar -C / -zx
+ARG TARGETARCH TARGETOS
```

Comment:
Leftovers from multi-arch experiments? Can't see that they are set anywhere.

Reply:
Exactly.

Reply:
Actually, just to check. Are you referring to `TARGET(OS|ARCH)` here?

If so, they are used on the following line.

Or did I misunderstand?

Reply:
Yes, as per our Slack conversation I didn't know that they were set automatically by buildkit. All good!

Reply:
Thanks for clarifying!